.devcontainer/devcontainer.json

```diff
@@ -59,12 +59,8 @@
   },
   "forwardPorts": [8000],
   "postCreateCommand": [
-    "poetry",
+    "pdm",
     "install",
```

Comment:
I think this is ok to just do `pdm install` because of the following from the [docs](https://pdm-project.org/en/stable/usage/dependency/#select-a-subset-of-dependency-groups-to-install)

> All development dependencies are included as long as --prod is not passed and -G doesn't specify any dev groups.

I think the [Litestar Contrib Guide](https://docs.litestar.dev/2/contribution-guide.html#running-the-docs-locally) is "wrong" (being explicit?)

> pdm install -G:docs

Reply:
that sounds good